src/gradle/orb.yml

```diff
@@ -0,0 +1,40 @@
+executor:
+  gradle:
+    description: The docker container to use when running Gradle builds
+    docker:
+      - image: 8-jdk-node
+    environment:
+      # Disable fancy terminal updates
+      TERM: dumb
+
+commands:
+  with_cache:
+    parameters:
+      steps:
+        type: steps
+    steps:
+      - run:
+          name: Generate Cache Checksum
+          commad: find . -name 'build.gradle' -exec cat {} + | shasum | awk '{print $1}' > /tmp/gradle_cache_seed
```

Comment:
typo: "commad"

Reply:
Also, I suggest putting in a comment to explain this line and why you aren't doing a checksum on build.gradle

Reply:
👍